src/implementations/vanilla/src/components/global-nav/top-nav/help/option/option.js

```diff
@@ -0,0 +1,25 @@
+import "./option.scss";
+
+const Template = require("./option.html");
+const Core = require("_core.js");
+const Interface = require("interface.json");
+
+/**
+ * Creates a Option
+ *
+ * @class
+ */
+class Option extends Core {
+  constructor(options) {
+    super(options);
+    this._render(Template, options);
+  }
+}
+
```

Comment:
Generally speaking defaults passed into the constructor have setters as well. After the refactor the react library may not even pass defaults.
-> So let's get setters in here for label, onClick, checked—any values used in the option component.

Also generally speaking, clickable things like buttons and options are implemented as `a` tags and have both `onClick` and `link` properties. We just don't know if the client will want to link away to a new page, or handle the click some other way. If they are building a big SPA, even links away will likely use `onClick` and be passed to the router rather than loading a new page. 
-> So rename `href` to `link` for consistency with other interfaces, and add an `onClick` method.

Reply:
Agreed on all points, updated in 3b0a0fe8